### ISSUE TITLE
[Bug]: No Idle Kick

### ISSUE THREAD

**Issue by Yimura** [Yimura/YimMenu#604]
### Describe the bug

No Idle kick does not appear to be working correctly.

### Steps To Reproduce

1. Enable "No Idle Kick"
2. Go AFK for 15 minutes
3. Get kicked

### Expected Behavior

Not be kicked.

### Store

Steam

### Additional context

Regression introduced in #588 

Fix offered by @maybegreat48 
```cpp
g_script_patcher_service->add_patch({ RAGE_JOAAT("freemode"), "2D 01 08 00 00 5D ? ? ? 56 ? ? 6E ", 5, { 0x2E, 0x01, 0x00 }, &g->tunables.no_idle_kick });
```

### Build

- [X] I used YimMenu unmodified

**Comment by ghost** [comment]
Did the global change with the halloween update? Maybe they changed their checks.

**Comment by Yimura** [comment]
Please check the PR I linked, you can see we're no longer using the global and instead patching the byte code of the script that handles the no idle kick.

**Comment by Yimura** [comment]
Fixed with #606
